jobs/config.json

```diff
@@ -597,6 +597,16 @@
       "sig-cluster-lifecycle"
     ]
   },
+  "ci-kubemci-image-push": {
+    "args": [
+      "make",
```

Comment:
I have no idea about how to configure this to run make using the Makefile in `images/kubemci/Makefile`?
Any pointers?

Reply:
`"make", "-C", "images/kubemci", "push"`

Reply:
less opaquely: `make` accepts `-C subdir` https://www.gnu.org/software/make/manual/make.html#Options-Summary

Reply:
Thanks for the super quick reply!!
Done